portals/application/configs/klear/model/DDIs.yaml

```diff
@@ -49,47 +49,52 @@ production:
             title: _("Unassigned")
             visualFilter:
               show: [ ]
-              hide: [ "IVRCommonId", "IVRCustomId", "huntGroupId", "userId", "faxId", "conferenceRoomId", "friendValue", "queueId"]
+              hide: [ "IVRCommonId", "IVRCustomId", "huntGroupId", "userId", "faxId", "conferenceRoomId", "friendValue", "queueId", "retailAccountId"]
           'user':
             title: _('user')
             visualFilter:
               show: ["userId" ]
-              hide: ["IVRCommonId", "IVRCustomId", "huntGroupId", "faxId", "conferenceRoomId", "friendValue", "queueId" ]
+              hide: ["IVRCommonId", "IVRCustomId", "huntGroupId", "faxId", "conferenceRoomId", "friendValue", "queueId", "retailAccountId" ]
           'IVRCommon':
             title: _('IVRCommon')
             visualFilter:
               show: [ "IVRCommonId" ]
-              hide: [ "userId", "IVRCustomId", "huntGroupId", "faxId", "conferenceRoomId", "friendValue", "queueId" ]
+              hide: [ "userId", "IVRCustomId", "huntGroupId", "faxId", "conferenceRoomId", "friendValue", "queueId", "retailAccountId" ]
```

Comment:
I also though many times but If we change this, we must also check Extension route selectbox. I'll give it a try.